.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+### Version
+<!-- The version of Nitrox you are using -->
+<!-- x.x.x.x -->
+
+### Bug
+<!-- Explain the bug that you have encountered -->
+
+### Expected
+<!-- Explain the expected behavior of the bug you encountered -->
+
+### Steps to reproduce
+<!-- Explain the steps to reproduce the encountered bug -->
+
+### Other details:
+<!-- Any other details that might help with this issue -->
+<!-- Check your player.log for any exceptions/errors that may have occurred (Especially if it's related to crashes or being unable to start Nitrox properly) and include them here -->
+<!-- Find your player.log located at %appdata%\.. then LocalLow\Unknowns Worlds\Subnautica e.g. C:\Users\Octocat\AppData\LocalLow\Unknown Worlds\Subnautica -->
```

Comment:
```suggestion
<!-- Find your player.log located at `%appdata%\..\LocalLow\Unknown Worlds\Subnautica` e.g. C:\Users\Octocat\AppData\LocalLow\Unknown Worlds\Subnautica -->
```

Reply:
Yeah you make a point here I did try to do this before but I must have made a mistake because it kept telling me that the path didn't exist. Thanks for this.